docs/index.md

```diff
@@ -37,7 +37,7 @@ Pulse provides push-based integrations with your workflow using the following al
 
     Pulse currently supports the following "one-click" integrations that simplify the process of setting up your workflows to send data to Pulse:
 
-    -   **[GitHub integration](one-click-integrations/github-integration.md):** reports changes and deployments
+    -   **[GitHub integration](one-click-integrations/github-integration.md) or [Bitbucket integration](one-click-integrations/bitbucket-integration.md):** reports changes and deployments
```

Comment:
I wonder if it may make sense to invert the fragments now that we have multiple items per line:
```suggestion
    -   **Report changes and deployments**: [GitHub integration](one-click-integrations/github-integration.md) or [Bitbucket integration](one-click-integrations/bitbucket-integration.md)
```